packages/btfs-http-client/js-btfs-common/master/js/protos/gogo_pb.js

```diff
@@ -0,0 +1,1993 @@
+// source: gogo.proto
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
```

Comment:
where did this get generated from? can we reference this as part of dloaddeps.sh?

Reply:
i will reference it from here https://github.com/TRON-US/protobuf/pull/3/files ,  when the dependency is available